isic_challenge_scoring/classification.py

```diff
@@ -31,7 +31,7 @@ def __init__(
         truth_probabilities: pd.DataFrame,
         prediction_probabilities: pd.DataFrame,
         truth_weights: pd.DataFrame,
-        validation_metric: Optional[ValidationMetric] = None,
+        target_metric: ValidationMetric = ValidationMetric.BALANCED_ACCURACY,
```

Comment:
Rename the `ValidationMetric` enum to `TargetMetric`?

Reply:
Actually, how about `ClassificationTargetMetric` or `ClassificationMetric`? I think we'll want a similar one for segmentation in the future.